docs/source/notebooks/wps_generate_climos_demo.ipynb

```diff
@@ -0,0 +1,402 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generate Climos"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This process runs [generate_climos](https://github.com/pacificclimate/climate-explorer-data-prep/blob/master/README.md#generate_climos-generate-climatological-means), which creates files with climatological means/standard deviations of input data from a netcdf file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from birdy import WPSClient\n",
+    "from pkg_resources import resource_filename\n",
+    "import requests\n",
+    "import os\n",
+    "from urllib.request import urlopen, urlretrieve\n",
+    "from datetime import datetime\n",
+    "from wps_tools.utils import copy_http_content\n",
+    "from netCDF4 import Dataset\n",
+    "from tempfile import NamedTemporaryFile\n",
+    "from bs4 import BeautifulSoup\n",
+    "import re"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Ensure we are in the working directory with access to the data\n",
+    "while os.path.basename(os.getcwd()) != \"thunderbird\":\n",
+    "    os.chdir('../')\n",
+    "\n",
+    "# Set up wps application\n",
+    "url = 'http://localhost:5001/wps'\n",
```

Comment:
I finally found the reason for the issues when running with the docker server url. It was nothing related to the docker configuration, and I realized the docker server can never read the files in local workstations.
```
annual_local = resource_filename('tests', 'data/gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100.nc')
```
@cairosanders could you relace the `annual_local` with an appropriate OPeNDAP file in THREDDS server and use the URL here?

Reply:
Good catch! And thanks for looking into this :smile: